### DISCUSSION TITLE
iSCSI loading in initramfs

### DISCUSSION THREAD

**Discussion by x3nomorpheus** [ltsp/ltsp#830] [Q&A]
Hi folk! I need to make my ltsp load squash image with iscsi instead of nfs. I setup the iscsi server, make a target with image of system. Rewrite ipxe conf for working with iscsi instead of nfs. But I had an error "_initiator reported error 15 session exists_". I think that it's happened because of double connection in init script located here **/scripts/local-top/iscsi** that operated by ltsp. If somebody has this error or know something about it, please could you help. I'm not sure that my diagnostic i fully right, so here is a line changed in ipxe conf for working with iscsi.
`set cmdline_method ip=dhcp root=iscsi:10.0.10.150::::iqn.2023-05.local.ltsp:jammy ro`

**Comment by alkisg** [comment]
Hi @x3nomorpheus, the `/scripts/local-top/iscsi` file is not part of LTSP.
It's part of the [open-iscsi](https://packages.ubuntu.com/jammy/amd64/open-iscsi/filelist) package, and it's handled by the initramfs-tools package.

LTSP gets loaded AFTER the iscsi mount succeeds, in initrd-bottom.
